tests/core/mempool/test_mempool_manager.py

```diff
@@ -111,12 +111,12 @@ def make_test_conds(
             Spend(
                 TEST_COIN.name(),
                 IDENTITY_PUZZLE_HASH,
-                uint32(height_relative) if height_relative else None,
+                None if height_relative is None else uint32(height_relative),
                 uint64(seconds_relative),
                 None,
                 None,
-                uint32(birth_height) if birth_height else None,
-                uint64(birth_seconds) if birth_seconds else None,
```

Comment:
the bug here is that a value of `0` is considered `False`, and will translate to `None`. The point of using an optional is to distinguish between a 0 and `None`